pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -628,9 +628,9 @@ message Image {
     // ID of the image.
     optional string id = 1;
     // Other names by which this image is known.
-    repeated string repo_tags = 2;
+    repeated string names = 2;
     // Digests by which this image is known.
-    repeated string repo_digests = 3;
+    repeated string digests = 3;
```

Comment:
I think we should also amend the description of [`ImageSpec`](https://github.com/kubernetes/kubernetes/blob/cc7f3906c7d03220459918ddb7485e575c6704aa/pkg/kubelet/api/v1alpha1/runtime/api.proto#L317) to be either `names` or `digests`